coreos/config/env/app-admin/setools

```diff
@@ -1,2 +1,2 @@
-export ac_cv_file__usr_lib64_libsepol_a=yes
+export ac_cv_file__build_amd64_usr__usr_lib64_libsepol_a=yes
```

Comment:
hm, this seems icky and will break for different arch/boards. I suppose it is ok for now to fix the build but something cleaner would be nice.